packages/create-svelte/templates/default/package.template.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "~TODO~",
 	"version": "0.0.1",
+	"private": true,
```

Comment:
should we do this in a separate PR ? .template.json won't be found by changesets

Reply:
good point, I hadn't spotted this. Agreed.